testing/helpers/cleanHtml.ts

```diff
@@ -10,7 +10,11 @@ export function cleanHtml(html: string, keepAttrs?: string[]): string {
   });
 
   // Condense whitespace:
-  html = html.trim().replace(/\s\s+/g, ' ');
+  html = cleanWhitespace(html);
 
   return html;
 }
+
+export function cleanWhitespace(html: string): string {
+  return html.trim().replace(/\s\s+/g, ' ');
+}
```

Comment:
Do we need this change?

Reply:
Extracting this as a separate function helped me write better test assertions.  I've got an assertion that looks like this:
```
expect(cleanHtml(el.outerHTML)).to.equal(cleanWhitespace(`
  <bento-mega-menu>
    <section ... etc .../>
  </bento-mega-menu>
`));
```